samples/client/petstore/php/SwaggerClient-php/lib/ObjectSerializer.php

```diff
@@ -162,6 +162,37 @@ public function toString($value)
     }
 
     /**
+     * Serialize an array to a string.
+     *
+     * @param array  $collection       collection to serialize to a string
+     * @param string $collectionFormat the format use for serialization (csv,
+     * ssv, tsv, pipes, multi)
+     *
+     * @return string
+     */
+    public function serializeCollection(array $collection, $collectionFormat, $allowCollectionFormatMulti=false)
+    {
+        if ($allowCollectionFormatMulti && ('multi' === $collectionFormat)) {
+            return http_build_query($collection, null, '&');
```

Comment:
FYI I just submitted a PR #1509 to support collectionFormat in Ruby client, which is using [Typhoeus](https://github.com/typhoeus/typhoeus) to make HTTP requests and it handles array parameters in the same way with the `multi` collectionFormat. So the [`build_collection_param`] method returns the (array) parameter directly.
I've no idea how PHP handles it, but hope this helps.